### DISCUSSION TITLE
Count of static pages

### DISCUSSION THREAD

**Discussion by eposha** [vercel/community#753] [Help] [answered]
Is there a limit to the number of static pages?

I use ISR on demand and fallback - blocking

A site could theoretically have several million pages?

Is it possible to store that many pages on Vercel?

**Comment by amyegan** [comment, marked as answer]
Hi @eposha. I'm not aware of any limits on number of pages, but there are other limits that effectively cap the number of pages a site can reasonably support. You can read more about these limits in our [limits doc](https://vercel.com/docs/concepts/limits/overview) and our [fair use policy](https://vercel.com/docs/concepts/limits/fair-use-policy). There are some strategies you can use to maximize what can be build within these limits, for example https://vercel.com/support/articles/what-can-i-do-when-i-run-into-build-output-limits-with-next-js-on-vercel

**Reply by eposha** [reply]
> Hi @eposha. I'm not aware of any limits on number of pages, but there are other limits that effectively cap the number of pages a site can reasonably support. You can read more about these limits in our [limits doc](https://vercel.com/docs/concepts/limits/overview) and our [fair use policy](https://vercel.com/docs/concepts/limits/fair-use-policy). There are some strategies you can use to maximize what can be build within these limits, for example https://vercel.com/support/articles/what-can-i-do-when-i-run-into-build-output-limits-with-next-js-on-vercel

Can help me with one more question, please

How I can computed or where I can see count of Edge Functions called?

**Reply by amyegan** [reply]
You can see [usage](https://vercel.com/docs/concepts/limits/usage) details for your account at https://vercel.com/dashboard/usage
